studio/components/to-be-cleaned/Storage/StorageExplorer/StorageExplorer.js

```diff
@@ -207,9 +206,6 @@ const StorageExplorer = observer(({ bucket }) => {
     if (selectedItemsToDelete.length === 1) {
       const [itemToDelete] = selectedItemsToDelete
       switch (itemToDelete.type) {
-        case STORAGE_ROW_TYPES.BUCKET:
```

Comment:
the very first UI had buckets within the explorer itself, i think i forgot to remove this